api/internal/target/kusttarget.go

```diff
@@ -113,7 +113,11 @@ func (kt *KustTarget) MakeCustomizedResMap() (resmap.ResMap, error) {
 }
 
 func (kt *KustTarget) makeCustomizedResMap() (resmap.ResMap, error) {
-	ra, err := kt.AccumulateTarget(&resource.Origin{})
+	var origin *resource.Origin
```

Comment:
nit: suggest using const var.

Reply:
The origin var changes as the resources are accumulated; the path and repo information get appended as we follow the kustomization tree, so I'm not sure how this can be a const

Reply:
weird, the comment is for "OriginAnnotations".

Reply:
Oh I see, I thought you meant for `var origin`. Anyways I changed `OriginAnnotations` to be const too. Thanks for the reviews!